modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -96,12 +96,6 @@ See  <<migrate,Migrate your platform>> for step-by-step instructions.
 == Constraints
 
 * If you have added custom indexes to certain tables in the Engine database, you must remove them before migrating to a later version. If you do not remove these indexes, the migration will not complete.
```

Comment:
them before migrating to a later version. => them before updating to a later version.

Reply:
the migration will not complete. => the update will not be complete.

Reply:
Maybe we should handle renaming of migration to update in another ticket since it's not related to what was done in this pr at all.

Reply:
And also, because there are 219 references to migration in this page alone.